package.json

```diff
@@ -9,7 +9,7 @@
     "clean": "rimraf lib",
     "precompile": "npm run clean",
     "compile": "cross-env NODE_ENV=production webpack --config ./config/webpack.prod.config.js --progress",
-    "start": "webpack-dev-server --config ./config/webpack.dev.config.js",
+    "start": "cross-env NODE_ENV=developmen webpack-dev-server --config ./config/webpack.dev.config.js",
```

Comment:
`NODE_ENV=development` i think

Reply:
if we are not using `webpack-dev-server` anymore, maybe let's remove it from the deps? Or is it still needed?

Reply:
we are still using it `cross-env NODE_ENV=development webpack-dev-server`

Reply:
oh woooooops!